### PR TITLE
fixed python2 bug faulty csv output

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: src/crate/crash/outputs.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
```
Comment (chaudum) [inline src/crate/crash/outputs.py]: still required?

File: src/crate/crash/outputs.py [hunk 2]
```diff
@@ -122,10 +124,13 @@ def json(self, result):
         yield self.to_json_str(obj)
 
     def csv(self, result):
-        wr = csv.writer(self.writer, doublequote=False, escapechar='\\')
+        wr = csv.writer(self.writer, doublequote=False, escapechar='\\', quotechar="'")
         wr.writerow(result.cols)
-        for row in result.rows:
-            wr.writerow(row)
+
+        json_dumps = lambda r: json.dumps(r, sort_keys = True)
```
Comment (chaudum) [inline src/crate/crash/outputs.py]: in the changes you write it is only required for python 2.x
then you could do a version check here

Comment (celaus) [inline src/crate/crash/outputs.py]: For output consistency I think it makes sense to do this in both versions.

**Comment by mikethebeer** [comment]
ack :santa:

**Pull request merged**